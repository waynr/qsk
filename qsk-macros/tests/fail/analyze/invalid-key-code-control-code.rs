use qsk_macros::layer;

fn main() {
    layer!(
        ModLayer: {
            F -> MEOW,
        },
    );
}
