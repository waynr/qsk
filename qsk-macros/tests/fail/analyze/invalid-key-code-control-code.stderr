error: invalid key code when converting parse::Key to ControlCode: VariantNotFound
 --> tests/fail/analyze/invalid-key-code-control-code.rs:6:18
  |
6 |             F -> MEOW,
  |                  ^^^^
